backend/locales/en-US.json

```diff
@@ -264,6 +264,15 @@
             "updating": "Updating"
         }
     },
+    "WarnThirdParty":{
+        "title_zip": "Third party plugin install",
+        "title_repo": "Third party store selection",
+        "button_processing_one": "Please wait {{timer}} second",
+        "button_processing_many": "Please wait {{timer}} seconds",
+        "button_idle": "I accept the risks",
+        "desc_zip": "This plugin that has been requested to be installed is taken outside our official store; we haven't audited it, so it will contain unvetted code; it might be doing exactly what it would be supposed to do or it might have additional features built in that will try to steal your data and take over your Steam account. The Decky team is not responsible for any problem arising from installing this plugin.",
```

Comment:
The following is less verbose while still conveying the same message. I don't want to have too much in this string, otherwise, people won't read it and it negates the whole point. This was approved by @AAGaming00 and reviewed by @TrainDoctor without any notes a week or two ago.

> The Decky Loader team has not reviewed this plugin. It may contain malware, such as software to steal your Steam account or harm your device. By installing this plugin, you agree you have assumed all risks to your device.